content/Foundation/Summit/2023/home.pml

```diff
@@ -35,49 +36,13 @@ Minneapolis, USA
-  </p>
-  <p class="keynote">
-    Accepting that the assurance effort should be commensurate with the criticality of the security product, my hope is that this presentation provides a sliding scale of seL4 features we should seek to take advantage of, be it for articulating an assurance case, or conducting an independent assessment.
-  </p>
-  <p style="margin-top:25px">
-   <img src="../../../images/summit/sam-security-photo.jpg"
-    style="width: 15%; float:right"
-    alt="Sam Leffler" />
-    <strong>Sam Leffler</strong> has worked at Google for nearly 15 years. He was part of the original team that developed ChromeOS and the ChromeBook, was responsible for the networking components of Project Loon, and worked on various infrastructure projects before joining Project Sparrow to help build the CantripOS embedded operating system. Prior to joining Google he was an independent contractor focused on wireless networking and operating systems. Before contracting he worked for VMware, Silicon Graphics, Pixar, Lucasfilm, and the Computer System Research Group (CSR) at UC Berkeley where he was responsible for the 4.2BSD release of the UNIX operating system. At CSRG he co-designed and implemented many of the facilities found in contemporary UNIX systems including sockets, networking support (IP/TCP), reliable signals, and more.
-  </p>
-
-  <p>
-    <strong>Sam's keynote will be about "CantripOS: An OS for Ambient ML Applications"</strong>
-  </p>
-  <h5 class="keynote" id="abstract-sam">Abstract</h5>
-  <p class="keynote" style="margin-bottom:35px">
-    CantripOS is an open source operating system purpose-built to run ML workloads for embedded systems. It is being developed as part of Google’s project Sparrow, whose charter is to build a low-power embedded platform for ML applications with a focus on security and privacy. CantripOS is built in the <a href="https://www.rust-lang.org/">Rust programming language</a>, runs under the <a href="https://sel4.systems/Info/Docs/seL4-manual-12.1.0.pdf">seL4 microkernel</a>, and uses a modified <a href="https://docs.sel4.systems/projects/camkes/">CAmkES framework</a>. This allows CantripOS to run dynamically loaded applications in a controlled sandbox, while still retaining the benefits of a statically designed system. This paper describes the system design, modifications made to seL4 and CAmkES, and future directions. All the work described here is publicly available on <a href="https://github.com/AmbiML/sparrow-manifest">GitHub</a>.
+    Have a look at the <a href="https://sel4summit2023.sched.com/">seL4 summit Program</a>!
```

Comment:
the program in Sched is still missing some talks (2nd keynote, panel..). We shouldn't publish before it's correct. I would also prefer if the default display is the grid one. It's a shame we can't have the 3 days side by side, or if not, have the full titles (space doesn't seem optimised). I'm not sure what's possible so if it's not possible, it's ok to ignore, but it'd be nice to have a "schedule at a glance" that people could look at without scrolling too much or click on stuff. Also: the colours hurt my eyes ;) Maybe we should provide Evi with the Foundation palette (can be found in the branding git repo) and we could have 2 blues for the talks (light/dark for short/standard talks), green for keynotes, yellow for discussions/announcements and light red for dinner/events...